packages/api/src/entities/User.ts

```diff
@@ -9,10 +9,19 @@ export type UserConstructorValues = ConstructorValues<User>;
 export class User extends Node<User> {
   @Property({ columnType: 'text' })
   name: string;
+  //pronouns: string;
+  //location: string;
+  //hireable: boolean;
+  //purpose: string;
+  //schoolName:string;
+  //major: string;
+  //graduationDate: Date
+
```

Comment:
Would this change be needed for the database side or would this be on the api PR #27 ?
How does typescript Date work with PostgreSQL date type?